src/main/resources/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder/config.jelly

```diff
@@ -305,6 +316,40 @@
                     success: function(data) {
                         mainTemplate = data;
 
+		     	//Handlebars is a kind of 'logic-less' like mustache, so it does not provide the equals operator
+		    	Handlebars.registerHelper('ifEquals', function(arg1, arg2, options) {
```

Comment:
Should we refactor these _Handlebars_ util functions into a separate JS file and import it here?